CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Kubermatic 2.20
 
+## [v2.20.2](https://github.com/kubermatic/kubermatic/releases/tag/v2.20.2)
```

Comment:
Let's maybe add a one-liner here to highlight that etcd is no longer a problem (since we discussed this problem in great detail for previous releases), like this:

```suggestion
## [v2.20.2](https://github.com/kubermatic/kubermatic/releases/tag/v2.20.2)

With this patch release, etcd for Kubernetes 1.22+ is upgraded to etcd 3.5.3. Data consistency issues as reported in previous release notes are fixed. Warnings and recommendations related to that can be considered withdrawn for this release.
```

Same goes for the other patch releases. WDYT?